spec/installed-package-view-spec.coffee

```diff
@@ -40,6 +40,10 @@ describe "InstalledPackageView", ->
     snippetsTable = null
     snippetsModule = null
 
+    # Relies on behavior not present in the snippets package before 1.33.
+    # TODO: These tests should always run once 1.33 is released.
+    shouldRunScopeTest = parseFloat(atom.getVersion()) >= 1.33
```

Comment:
This is slightly horrifying but I'm surprised to find that it works pretty well for this purpose even with versions like `1.33.0-beta3` :)  Since we're using the pattern in other repos I think it's fine to use it here too.

Reply:
> Huh I'm getting this error when trying to test out the changes. I don't know why I definitely pulled and linked both the settings-view and snippets pr. I'll check more tomorrow.

I bet that's a legit error. It occurred to me that I didn’t actually test out the functionality after that change — I only ran the specs. But the specs use a mock snippets provider, so I haven't tested the integration at all. I'll look at that shortly.

Reply:
Haha, yeah. We get many commenting about this. And also the fact that 1.5 > 1.33. But we still use it :)